backend/src/ee/services/permission/project-permission.ts

```diff
@@ -339,6 +459,17 @@ const buildAdminPermissionRules = () => {
     );
   });
 
+  can(
+    [
+      ProjectPermissionDynamicSecretActions.Read,
+      ProjectPermissionDynamicSecretActions.Edit,
```

Comment:
nit: Since this is the first time we are creating some of these subjects, i think it's worth making the action names more in line with what they will do. Similar to `lease`.

Reply:
todo: Akhi will rename the actions